client/src/components/CollapsePanel/index.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Collapse } from 'antd';
+
+import './index.css';
+
+const { Panel } = Collapse;
+
+const renderTag = (content, className) => {
+  const TagName = content.tag;
+  return content.values.map((element, index2) => (
+    <TagName key={`${TagName}${index2 + 1}`} className={className}>
+      {element}
+    </TagName>
+  ));
+};
+
+const renderParentTag = (content, classParent, classChild, keyParent) => {
+  const ParentName = content.parent;
+  const TagName = content.tag;
+  return (
+    <ParentName key={keyParent} className={classParent}>
+      {content.values.map((element, index1) => {
+        return (
+          <TagName key={`${TagName}${index1 + 1}`} className={classChild}>
+            {element}
+          </TagName>
+        );
+      })}
+    </ParentName>
+  );
+};
+
+const renderImgTag = (content, className) => (
+  <img
+    key={content.value}
+    className={className}
+    src={content.value}
+    alt="images question"
+  />
+);
+
+const renderChild = (content, classParent, classChild, keyParent) => {
+  const ChildName = content.child.tag;
+  const TagName = content.tag;
+  return (
+    <TagName key={keyParent} className={classParent}>
+      {content.value}
+      <ChildName className={classChild}>{content.child.value}</ChildName>
+    </TagName>
+  );
+};
+
+const renderQuestion = (question, objStyle) =>
+  question.content.map((content, index) => {
+    const TagName = content.tag;
+    const classTag = objStyle[`${TagName}Class`];
+    if (content.parent) {
+      const ParentName = content.parent;
+      const classParent = objStyle[`${ParentName}Class`];
+      const keyParent = ParentName + index + 1;
+      return renderParentTag(content, classParent, classTag, keyParent);
+    }
+    if (content.tag === 'img') {
+      return renderImgTag(content, classTag);
+    }
+    if (content.child) {
+      const ChildName = content.child.tag;
+      const classChild = objStyle[`${ChildName}Class`];
+      const keyParent = TagName + index + 1;
+      return renderChild(content, classTag, classChild, keyParent);
+    }
+    return renderTag(content, classTag);
+  });
+
+const renderSubQuestions = (question, objStyle, index) => {
```

Comment:
btw now I realized that this is a generic `CollapsePanel` component, i.e. it has nothing to do with questions (it is only a coincidence that it is used by FAQ, it could be used by other components).. So this is a wrong name

Reply:
It is gonna be used by another component, apparently the code in this PR is related to his next PR, which is #59

Reply:
that doesn't matter, this component stands on its own and as such doesn't know that it is used for FAQ (and again, can be used for other things..). so the name is not correct